docs/docs/reference/metaprogramming/macros.md

```diff
@@ -192,8 +192,8 @@ quote but no splice between the parameter binding of `T` and its
 usage. But the code can be rewritten by adding a binding of a `Type[T]` tag:
 
 ```scala
-def to[T, R](f: Expr[T] => Expr[R])(using Type[T], Type[R], Quotes): Expr[T => R] =
-   '{ (x: T) => ${ f('x) } }
+def to[T, R](f: Expr[T] => Expr[R])(using t: Type[T])(using Type[R], Quotes): Expr[T => R] =
+   '{ (x: $t) => ${ f('x) } }
 ```
 
 In this version of `to`, the type of `x` is now the result of
```

Comment:
```
In this version of `to`, the type of `x` is now the result of inserting the type `Type[T]` and selecting its `Underlying `. 
To avoid clutter the compiler converts any type reference to a type `T` in subsequent phases to `summon[Type[T]].Underlying`.
And to avoid duplication it does it once per type and creates an alias for that type at the start of the quote.
For instance, the user-level definition of `to`:
```

And also change bellow
```diff
-   '{ (x: t.Underlying) => ${ f('x) } }
+   '{ 
+      type T = t.Underlying
+      (x: T) => ${ f('x) } 
+.   }
```

Reply:
Addressed